docs/en/02_Developer_Guides/03_Forms/Field_types/03_HTMLEditorField.md

```diff
@@ -224,9 +224,9 @@ Example: Remove field for "image captions"
 
 	:::php
 	// File: mysite/code/MyToolbarExtension.php
-	class MyToolbarExtension extends Extension {
+	class MyToolbarExtension extends DataExtension {
```

Comment:
It is in the API docs (e.g. start typing `extension::add_to_class`, but it’s hard to find as it comes after `DataExtension` alphabetically and we only show a certain number of results :/